README.md

```diff
@@ -43,13 +44,15 @@ docker pull ghcr.io/thorpejosh/truenas-zfs-unlock:latest
 ```
 
 ## How to use this image
-This tool is designed to be run on a machine that has network access to your Truenas server but preferably not in the same location. It can run on a cloud server, raspberry pi, laptop, workstation, etc.
+This tool is designed to run on a machine that has network access to the Truenas server but preferably not in the same physical location. This image can run on a cloud server, raspberry pi, laptop, workstation, etc.
 
-Firstly setup (or edit) your zfs datasets to use "Passphrase" encryption so that you set the key yourself and it won't be stored by Truenas. Use a password generator and make it long (128 characters for example).
+Firstly set up (or edit) your zfs datasets to use "Passphrase" encryption, this will enable you to set the key yourself and therefore the encryption keys won't be stored by Truenas.
+
+Use a password generator that is set to use capital and lowercase lettering, numerals and special characters with the maximum allowable length (128 characters for example).
```

Comment:
This is very chatGPT like response. I'd probably delete this line completely. No need to explain to someone how to generate a password.